asgard/direct_path_response_builder.cpp

```diff
@@ -191,15 +203,18 @@ void set_path_item_type(const TripLeg_Edge& edge, pbnavitia::PathItem& path_item
     }
 }
 
-// The duration of the path_item is the current node duration - previous_node_elapsed_time
-// Returns node duration if current node has one, else returns 0
-uint32_t set_path_item_duration(const TripLeg_Node& node, uint32_t previous_node_elapsed_time, pbnavitia::PathItem& path_item) {
-    if (node.has_elapsed_time()) {
-        path_item.set_duration(node.elapsed_time() - previous_node_elapsed_time);
-        return node.elapsed_time();
+void set_path_item_duration(const DirectionsLeg_Maneuver& maneuver, pbnavitia::PathItem& path_item) {
+    if (maneuver.has_time()) {
+        path_item.set_duration(maneuver.time());
     }
+}
 
-    return 0;
+void set_path_item_direction(const DirectionsLeg_Maneuver& maneuver, pbnavitia::PathItem& path_item) {
+    if (maneuver.has_turn_degree()) {
+        int turn_degree = maneuver.turn_degree() % 360;
+        if (turn_degree > 180) turn_degree -= 360;
```

Comment:
It's really minor but

```suggestion
        if (turn_degree > 180) {
            turn_degree -= 360;
        }
```
To be coherent with the navitia coding style and to avoid classic bugs if you add a new line.